test/e2e/icon.test.js

```diff
@@ -9,9 +9,40 @@ describe('vl-icon', async () => {
         return vlIconPage.load();
     });
 
-    it('placeholder', async () => {
-        assert.isTrue(true);
+    it('als gebruiker wil ik het verschil kunnen zien tussen een klein en een gewoon icon', async () => {
+        const icon = await vlIconPage.getSmallIcon();
+        await assert.eventually.isTrue(icon.isSmall());
```

Comment:
Voor de zekerheid overal de ontkennende assert toevoegen voor het gewoon icoon, zo zijn we zeker dat we geen false positive asserts hebben.

Reply:
https://media.giphy.com/media/DRUBFO5WTwppu/giphy.gif

Reply:
![guido](https://user-images.githubusercontent.com/11139249/69817204-614aa700-11fa-11ea-9525-7fff259aac3f.gif)